tex/measure/lebesgue-int.tex

```diff
@@ -147,6 +177,69 @@ \section{The definition}
 as we won't really have a use for them in the context of probability.
 (They appear later on in a bonus chapter.)
 
+\section{An equivalent definition}
```

Comment:
approach used from Wheeden. In chapter 1-9 of that textbook before ``abstract integration'', the only measure space that is used is ℝ^n, which means the measure over Ω × ℝ is obtained ``for free''.

This definition also allows proving the monotone convergence theorem really easily.